### PR TITLE
Enable favorite item's context menu on Tablet.

### PR DESCRIPTION
**Pull request by rojkov**

### CODE REVIEW

**Comment by tworaz** [comment]
LGTM

File: src/pages/components/FavoriteContextMenu.qml [hunk 1]
```diff
@@ -20,6 +20,9 @@ Component {
         property string title
         property string url
 
+        // FavoriteGrid doesn't fill the entire page thus set width to FavoriteGrid's parent
+        width: _flickable !== null ? _flickable.parent.width : (parent ? parent.width : 0)
+
```
Comment (rainemak) [inline src/pages/components/FavoriteContextMenu.qml]: Menu position is not updated correctly when rotating the device so that context menu is open.

Comment (rojkov) [inline src/pages/components/FavoriteContextMenu.qml]: Updated PR:
If the binding for x is not broken (see below) then the position is correct upon orientation change.

**Comment by rainemak** [comment]
LGTM. Looking good!

**Pull request merged**